bitrot.go

```diff
@@ -219,6 +225,37 @@ func (cmd *Compare) Execute(args []string) (err error) {
 	return nil
 }
 
+func (cmd *CompareLatestManifests) Execute(args []string) (err error) {
+	assertNoExtraArgs(&args, cmd.logger)
+	oldPath := pathString(cmd.Arguments.Old)
+	newPath := pathString(cmd.Arguments.New)
+
+	oldManifest := LatestManifestFileForPath(oldPath)
+	newManifest := LatestManifestFileForPath(newPath)
+
+	if oldManifest == nil {
```

Comment:
But, I think I want to postpone that and fix all the string literals later on. Lots of duplication there already.